syntax="proto3";

package pb;

option go_package = "../pb";

message ImageRequest {
	bytes Image = 1;
}


message ImageResponse {
	bytes ProcessedImage = 1;
}


service Remove {
	
	rpc RemoveBG(stream ImageRequest) returns (ImageResponse) {}
}
